apisix/plugins/jwt-auth.lua

```diff
@@ -407,6 +434,25 @@ function _M.rewrite(conf, ctx)
         return 401, {message = "failed to verify jwt"}
     end
 
+    -- check for hiding `Authorization` request header if `hide_credentials` is `true`
+    if conf.hide_credentials then
+        -- hide sensitive field
+        if from_header then
+            -- hide for header
+            core.request.set_header(ctx, conf.header, nil)
+
+        elseif from_query then
+            -- hide for query
+            local args = core.request.get_uri_args(ctx)
+            args[conf.query] = nil
+            core.request.set_uri_args(ctx, args)
+
+        else
+            -- hide for cookie
+            set_our_cookie(conf.cookie, "deleted; Max-Age=0")
```

Comment:
I think it's not a good way to hide the cookie.

Reply:
I use the [lua-resty-cookie](https://github.com/cloudflare/lua-resty-cookie) lib to update cookie this time.